native/src/utils/loadResourceCache.ts

```diff
@@ -51,8 +51,18 @@ const loadResourceCache = async ({
     return
   }
 
-  const fetchMap = resourceURLFinder.buildFetchMap(input, (url, urlHash) =>
-    buildResourceFilePath(url, cityCode, urlHash),
+  const currentResourceCache = await dataContainer.getResourceCache(cityCode, languageCode)
+
+  const currentlyCachedFiles: string[] = []
+  Object.values(currentResourceCache).forEach(filesObject => {
+    const fileUrls = Object.keys(filesObject)
+    fileUrls.forEach(url => currentlyCachedFiles.push(url))
+  })
```

Comment:
:wrench: 
```suggestion
  const currentlyCachedFiles = Object.values(currentResourceCache).map(Object.keys).flat()
```

Reply:
Woooooow, that is some serious JS magic! Thanks!